projects/aca-folder-rules/src/lib/rule-details/actions/rule-action.ui-component.ts

```diff
@@ -74,13 +74,13 @@ export class RuleActionUiComponent implements ControlValueAccessor, OnInit, OnDe
     this.setDisabledState(isReadOnly);
   }
 
-  private _aspects;
+  private _parameterConstraints;
```

Comment:
maybe should be initialised to an empty array by default, not to crash on the first `get` without `set`

Reply:
Done, ty